conformance/lang/expressions/unary-expression/unary-complement-expression.balt

```diff
@@ -0,0 +1,515 @@
+Test-Case: output
+Description: Test unary complement basic operations.
+Labels: unary-complement, HexIntLiteral
```

Comment:
Need to include `int` as a label.

Applicable in other places too. `int`, `DecimalNumber`, or `HexIntLiteral`.